smart-frontend/app/src/components/uikit/DateRangePicker/DateRangePicker.constants.ts

```diff
@@ -0,0 +1,15 @@
+import type { RangePreset } from './DateRangePicker.types';
+
+export const defaultRangesPreset: RangePreset[] = [
+  { id: 'now-1h', description: 'last 1 hour' },
+  { id: 'now-2h', description: 'last 2 hours' },
+  { id: 'now-4h', description: 'last 4 hours' },
+  { id: 'now-8h', description: 'last 8 hours' },
+  { id: 'now-12h', description: 'last 12 hours' },
+  { id: 'now-24h', description: 'last 24 hours' },
```

Comment:
I think we can take last @remizov-arena changes from adcc

Reply:
I think no. I change default presets for ADCC, because it was agreed upon with Voronkov. But presets from this MR uses in ADQM, and it's correct for ADQM bussines issues. 
We change this presets in every time in future